ReactWindows/ReactNative.Shared/UIManager/ViewManagerBase.cs

```diff
@@ -18,8 +14,7 @@ namespace ReactNative.UIManager
     /// <see cref="ReactShadowNode"/> subclasses used for calculating position
     /// and size for the corresponding native view.
     /// </summary>
-    public abstract class DependencyObjectViewManager<TDependencyObject, TReactShadowNode> : IViewManager
-        where TDependencyObject : DependencyObject
+    public abstract class ViewManagerBase<TView, TReactShadowNode> : IViewManager
```

Comment:
So we have  ViewManagerBase and per flavor BaseViewManager now :)

Reply:
Sadly `ViewManager` is already taken. That would be a good name here. I suck at naming, now is the time to make the call so if you have something better, please suggest :-)